docs/azure/sdk/media/mermaidjs/authentication-environments.md

```diff
@@ -0,0 +1,47 @@
+---
+ms.topic: include
+ms.date: 08/07/2024
+---
+
+```mermaid
+%% STEPS TO GENERATE IMAGE 
+%% ======================= 
+%% 1. Install mermaid CLI v10.9.1 (see https://github.com/mermaid-js/mermaid-cli/blob/master/README.md): 
+%%    npm i -g @mermaid-js/mermaid-cli@10.9.1 
+%% 2. Run command: mmdc -i authentication-environments.md -o ../../media/mermaidjs/authentication-environments.svg 
+
+%%{init: {'theme':'base', 'themeVariables': { 'primaryColor': '#fff', 'edgeLabelBackground':'#fff', 'fontSize': '24px'}}}%%
+flowchart LR
+    NetApp[".NET app"]
+    Q1{Where is the app running?}
+    
+    NetApp --> Q1
+    
+    %% Local Development Machine Branch
+    Q1 --> LocalDev[Development machine]
+    LocalDev --> AppSP["**Service principal**"]
+    LocalDev --> DevAccount["**Developer account**"]
+    LocalDev --> Broker["**Broker**"]
```

Comment:
I think we are mixing concepts here - Broker would be a developer account also. Should we use only `Service principal` and `User principal`? We could probably get rid of Broker.

Reply:
The idea behind this diagram is the authentication approach or technique that the user would implement, so even though the underlying account is the same its acquired using a different approach.

Reply:
I think Chris' suggestion here makes a lot of sense. Please make that change in this diagram.